test/unit/tags/breadcrumbs.ts

```diff
@@ -23,55 +23,62 @@ suite('gb-breadcrumbs', Breadcrumbs, ({
 
   describe('clearRefinements()', () => {
     it('should update refinements with empty array', () => {
-      const updateRefinements = stub(tag(), 'updateRefinements');
-
+      spy = tag().update = sinon.spy((obj) => expect(obj).to.eql({ selected: [] }));
```

Comment:
the `spy` that's exposed is actually a shortform for `sandbox().spy` so it can be used as a function